### PR TITLE
:sparkles: string to regexp

### LINKED ISSUE #10: add path-to-regexp support
Link : https://github.com/pillarjs/path-to-regexp
Source : https://github.com/mobxjs/mobx-state-tree/issues/486#issuecomment-339965337

### PR DESCRIPTION
**Pull request by fabienjuif**
fixes #10

### CODE REVIEW

**Comment by fabienjuif** [comment]
What do you think of this @wbercx ? 
This is about the path to regexp support :)

File: README.md [hunk 1]
```diff
@@ -84,7 +84,9 @@ Note that:
 ### Take API
 `take` is an helper that takes two arguments (`take(test, reaction)`):
  - first argument is the `test`, it can be
-   * **a string:** then the fullpath equality is tested
+   * **a string:** this string will be converted to a regular expression then the match is tested with fullpath
+     - `'/user/add'` will work against `'/user/add/`
```
Comment (delphinemillet) [inline README.md]: missing closing quote (on the 2 lines)

File: src/helpers/take.js [hunk 2]
```diff
@@ -1,10 +1,20 @@
 import { isRegExp, isString, isFunction } from 'lodash'
+import pathToRegexp from 'path-to-regexp'
 
 const take = (match, callback) => (action, tree) => {
   const isMatching = false
```
Comment (delphinemillet) [inline src/helpers/take.js]: this test confuses me : at a first fast read, I thought it was an affectation, then I had to think why this test. According to me, it slows the reading

**Review by delphinemillet — COMMENTED** [review]
* typos on readme
* code style to discuss

**Review by delphinemillet — APPROVED** [review]

**Pull request merged**